src/html5_parser/parser/document.rs

```diff
@@ -109,93 +109,3 @@ impl fmt::Display for Document {
-        );
-        let title_id = document.add_node(
-            super::Node::new_element("title", HashMap::new(), HTML_NAMESPACE),
-            head_id,
-        );
-        let title_text_id = document.add_node(super::Node::new_text("Hello world"), title_id);
-        let p_id = document.add_node(
-            super::Node::new_element("p", HashMap::new(), HTML_NAMESPACE),
-            body_id,
-        );
-        let p_text_id = document.add_node(super::Node::new_text("This is a paragraph"), p_id);
-        let p_comment_id = document.add_node(super::Node::new_comment("This is a comment"), p_id);
-        let p_text2_id =
-            document.add_node(super::Node::new_text("This is another paragraph"), p_id);
-        let p_text3_id =
-            document.add_node(super::Node::new_text("This is a third paragraph"), p_id);
-        let p_text4_id =
-            document.add_node(super::Node::new_text("This is a fourth paragraph"), p_id);
-        let p_text5_id =
-            document.add_node(super::Node::new_text("This is a fifth paragraph"), p_id);
-        let p_text6_id =
-            document.add_node(super::Node::new_text("This is a sixth paragraph"), p_id);
-        let p_text7_id =
-            document.add_node(super::Node::new_text("This is a seventh paragraph"), p_id);
-        let p_text8_id =
-            document.add_node(super::Node::new_text("This is a eighth paragraph"), p_id);
-        let p_text9_id =
-            document.add_node(super::Node::new_text("This is a ninth paragraph"), p_id);
-
-        document.append(p_text9_id, p_id);
-        document.append(p_text8_id, p_id);
-        document.append(p_text7_id, p_id);
-        document.append(p_text6_id, p_id);
-        document.append(p_text5_id, p_id);
-        document.append(p_text4_id, p_id);
-        document.append(p_text3_id, p_id);
-        document.append(p_text2_id, p_id);
-        document.append(p_comment_id, p_id);
-        document.append(p_text_id, p_id);
-        document.append(p_id, body_id);
-        document.append(title_text_id, title_id);
-        document.append(title_id, head_id);
-        document.append(head_id, html_id);
-        document.append(body_id, html_id);
-        document.append(html_id, root_id);
-
-        assert_eq!(
-            format!("{}", document),
-            r#"Document
-        └─ <html>
-            └─ <head>
-                └─ <title>
-                └─ Hello world
-            └─ <body>
-                └─ <p>
-                └─ This is a paragraph
-                └─ <!-- This is a comment -->
-                └─ This is another paragraph
-                └─ This is a third paragraph
-                └─ This is a fourth paragraph
-                └─ This is a fifth paragraph
-                └─ This is a sixth paragraph
-                └─ This is a seventh paragraph
-                └─ This is a eighth paragraph
-                └─ This is a ninth paragraph
-        "#
-        );
-    }
-}
```

Comment:
These tests would be reintroduced once they're all passing on a topic branch.

Reply:
i agree.. I'm currently trying to fix those tests, but we can remove them for now.  My goal is indeed a stable/compilable main branch